forum/qa-include/pages/question-view.php

```diff
@@ -697,11 +697,13 @@ function qa_page_q_comment_follow_list($question, $parent, $commentsfollows, $al
 				$showcomments[$commentfollowid]=$commentfollow;
 
 		$countshowcomments=count($showcomments);
-
-		if ( (!$alwaysfull) && ($countshowcomments > qa_opt('show_fewer_cs_from')) )
+		/* Show all comments */
+		$skipfirst = 0;
+		/* ----------------- */
+		/*if ( (!$alwaysfull) && ($countshowcomments > qa_opt('show_fewer_cs_from')) )
 			$skipfirst=$countshowcomments-qa_opt('show_fewer_cs_count');
 		else
-			$skipfirst=0;
+			$skipfirst=0;*/

```

Comment:
Komentowany kod to morderczy kod. Za rok tu spojrze i bede się zastanawiać czy przypadkiem nie odkomentować